packages/workflow/src/records/handler/create.ts

```diff
@@ -8,39 +8,50 @@
  *
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
-import * as Hapi from '@hapi/hapi'
+
```

Comment:
These changes are the most significant in this PR so please read **extremely carefully**, compare to the old implementation and make parts of the flows fail intentionally by adding for instance

```
if(Math.random() > 0.00005) {
  throw new Error('Random error!')
}
```

To see current status of BullMQ, open up the dashboard by running

```
docker run -p 3003:3000 -e BULLMQ_DASHBOARD=true taskforcesh/bullmq-dashboard
```

and then opening up http://localhost:3003